config/environments/test.rb

```diff
@@ -78,4 +78,9 @@
 
   # Switch to determine whether or not o collect HMRC data
   config.x.collect_hmrc_data = true
+
+  config.after_initialize do
+    Prosopite.rails_logger = true
+    Prosopite.raise = true
```

Comment:
I'd expect us to disable this initially if there are a significant number of failures, and devise a strategy to incrementally remove any N+1 queries.